data.json

```
@@ -163,6 +163,11 @@
                     "value": "https://ci.lucko.me/view/LuckPerms/"
                 }
             ]
+        },
+            "command": "paste",
+            "title": "PasteBin",
+            "url": "https://pastebin.com/",
+            "description": "We don't like seeing chat flooded with error codes... Paste your error codes or configuration files in here and share the link!",
```

That spelling sounds a bit annoyed/angry.
It's not like we don't like people sharing errors through Discord, but that hastebin, Pastebin, etc. are a better alternative to more easily read the text.

Also, you can add alternative links to the response.
```suggestion
            "description": "Hastebin allows to easily share config files or errors without making a mess.\n**Please make sure to remove passwords, usernames, etc. when sharing config files or similar!**",
            "fields": [
                {
                    "key": "Pastebin",
                    "value": "https://pastebin.com/"
                },
                {
                    "key": "Paste.helpch.at",
                    "value": "https://paste.helpch.at"
                }
            ]
```